### DISCUSSION TITLE
Automation account naming

### DISCUSSION THREAD

**Discussion by Asha-Hiremath** [Azure/Enterprise-Scale#782] [Q&A] [answered]
Is it a spell mistake as there is a double "a" infront of aauto. 

![image](https://user-images.githubusercontent.com/79122967/131938525-ebd4f233-02ad-449c-b7f4-fb1903aa15be.png)

Also please advise how to run the eslzARM powershell commands in Devops as a pipeline?

**Comment by krnese** [comment, marked as answer]
See details around this here: https://github.com/Azure/Enterprise-Scale/blob/main/docs/EnterpriseScale-FAQ.md#can-we-take-the-arm-templates-for-enterprise-scale-reference-implementations-and-check-them-into-our-repository-and-deploy-it-from-there-instead-of-via-the-azure-portal

The gist of it is that we recommend using AzOps (you can integrate during deployment - as well as any time post deployment and reconcile the templates back into Git).

If you want to take the templates as is in a 'do it yourself' fashion, we have documented the strict sequence and deployment steps here: https://github.com/Azure/Enterprise-Scale/tree/main/eslzArm

Regarding the suffix of the automation account, this is not a mistake, but by design with regards to min character lenght for the automation account resourceType
